### PR TITLE
Feature/overview

### PR DESCRIPTION
**Pull request by van-hsieh**
Hi Fernanda, 

Can you review my PR for recording progress demo? This is the basic prototype with all the React functionalities minus dynamic UI changes.

### CODE REVIEW

File: client/src/components/Overview/subcomponents/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Category = ({ product }) => (
+  <div>
+    <div>{product.category}</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/Category.jsx]: I would suggest to use text html tags like `<p>` or `headers(<h1>)` instead using `<div>`

File: client/src/components/Overview/subcomponents/Description.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Description = ({ product }) => (
+  <div>
+    <div>{product.slogan}</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/Description.jsx]: same as above. Let's use text html tags.

File: client/src/components/Overview/subcomponents/DropDown.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  closeMenu() {
+    this.setState({ showMenu: false }, () => {
+      document.removeEventListener('click', this.closeMemu);
+    });
+  }
+
+  toggleMenu(e) {
+    console.log('ss')
+    e.stopPropagation();
+    e.nativeEvent.stopImmediatePropagation();
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/DropDown.jsx]: I'm seeing this error on the browser when clicking on the button to toggle photos, sizes or quantity
image

File: client/src/components/Overview/subcomponents/Title.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Title = ({ product }) => (
+  <div>
+    <div>{product.name}</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/Title.jsx]: same as above. Let's use text html tags.

File: client/src/components/Overview/subcomponents/StyleSelector.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import css from '../styles/styleSelector.module.css';
+
+const StyleSelector = ({ styles, currStyle, handleStyleSelect }) => (
+  <div>
+    <div><b>STYLE {'>'} </b> {currStyle.name}</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/StyleSelector.jsx]: same as above. Let's use text html tags.

File: client/src/components/Overview/subcomponents/SizeSelector.jsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    .filter(sku => currStyle.skus[sku].quantity !== 0);
+  const availableSizes = {};
+  for (var sku of availableSkus) {
+    availableSizes[sku] = currStyle.skus[sku].size;
+  }
+
+  return (
+    <ul>
+      <Dropdown options={availableSizes} clickHandler={handleSizeSelect}/>
+      {/* {(isNoAvailble(currStyle)) ?
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/SizeSelector.jsx]: Why is this commented?

File: client/src/components/Overview/subcomponents/Ratings.jsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+const Ratings = ({ reviews }) => {
+  const length = reviews.length;
+  const total = reviews.reduce((acc, review) => {
+    return acc + review.rating;
+  }, 0);
+  const average = total / length;
+
+  return (
+    <div>
+      <div>rating: {average}</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/Ratings.jsx]: same as above. Let's use text html tags.

File: client/src/components/Overview/subcomponents/Price.jsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Price = ({ currStyle }) => (
+  <div>
+    <div>${ currStyle.original_price }</div>
```
Comment (fernanda1411) [inline client/src/components/Overview/subcomponents/Price.jsx]: same as above. Let's use text html tags.

**Review by fernanda1411 — CHANGES_REQUESTED** [review]
Great Job Van! 
I suggest to add a blank line at the end of files and avoid having more than one blank line in between code blocks. Also let's use text html tags when dealing with text.

File: client/src/components/Overview/subcomponents/SizeSelector.jsx (hunk 6 above)
Comment (van-hsieh) [reply client/src/components/Overview/subcomponents/SizeSelector.jsx]: Kept if for further refactoring. Current drop down if half baked. Removed from this PR to keep our main code base clean. Thanks for pointing it out.

**Comment by van-hsieh** [comment]
Hi Nanda, thanks a bunch for the review. I updated base on your comment and merged.

**Pull request merged**